### PR TITLE
Fixes #1942 added campaigns to course page

### LINKED ISSUE #1942: Campaign should be included in course pages on-wiki
Hello, 

it will be great if campaign(s) can be passed to the course templates. Cs.wikipedia wants to have them displayed. 

Thanks!

Best,
Martin Urbanec

### PR DESCRIPTION
**Pull request by garry329**
I have added the names of all campaigns that a course is part of to be shown on courses page.

### CODE REVIEW

**Comment by garry329** [comment]
Ok @ragesoss !!Will do that..

File: spec/lib/wiki_course_output_spec.rb [hunk 1]
```diff
@@ -44,6 +44,22 @@
                       timeline_start: '2016-01-11',
                       timeline_end: '2016-04-24',
                       weeks: [week1, week2])
+      campaign1 = create(:campaign,
+                         id: 1,
```
Comment (ragesoss) [inline spec/lib/wiki_course_output_spec.rb]: These IDs are not necessary. If you leave them out, IDs will be generated automatically.

Comment (garry329) [reply spec/lib/wiki_course_output_spec.rb]: Ok...is the rest ok?

Comment (ragesoss) [reply spec/lib/wiki_course_output_spec.rb]: tes, it looks reasonable.

Comment (garry329) [reply spec/lib/wiki_course_output_spec.rb]: @ragesoss I have removed the ids in the latest commit...

Comment (garry329) [reply spec/lib/wiki_course_output_spec.rb]: @ragesoss can you suggest me a rails issue i can try to solve next?

Comment (garry329) [reply spec/lib/wiki_course_output_spec.rb]: @ragesoss can you please tell me why this failing in travis?
`rspec ./spec/features/admin_role_spec.rb:144 # Admin users linking a course to its Salesforce record makes the Link to Salesforce button appear`

Comment (garry329) [reply spec/lib/wiki_course_output_spec.rb]: I haven't made any changes around this....

Comment (ragesoss) [reply spec/lib/wiki_course_output_spec.rb]: That's a test that occasionally fails randomly. I've restarted the build.

Comment (garry329) [reply spec/lib/wiki_course_output_spec.rb]: @ragesoss i am facing another issue which is unrelated..
`rspec ./spec/features/multiwiki_assignment_spec.rb:69 # multiwiki assignments will create a valid assignment for multilingual wikisource projects`

**Pull request merged**